README.md

```diff
@@ -1,9 +1,992 @@
-# Splunk> Phantom
+[comment]: # "Auto-generated SOAR connector documentation"
```

Comment:
Can we update this auto generated content? It still references Phantom

Reply:
We have added [comment]: # "Auto-generated SOAR connector documentation" in place of # Splunk> Phantom, hence Phantom won't be visible.